README.md

```diff
@@ -312,64 +312,64 @@ print(get_all_alerts())
 
 ## Synchronizing external SIEM
 
-In addition to `Alerts`, Cognitive Intelligence API also offers more granular information like `ThreatOccurrences` or `Sightings`.
+In addition to `Alerts`, Global Threat Alerts API also offers more granular information like `ThreatDetections` or `Events`.
 
-> Learn more about `ThreatOccurrence` and `Sighting` in
->[Cognitive Intelligence Documentation](http://www.cisco.com/c/en/us/td/docs/security/web_security/scancenter/administrator/guide/b_ScanCenter_Administrator_Guide/b_ScanCenter_Administrator_Guide_chapter_011110.html)
+> Learn more about `ThreatDetection` and `Event` in
+>[Global Threat Alerts Documentation](http://www.cisco.com/c/en/us/td/docs/security/web_security/scancenter/administrator/guide/b_ScanCenter_Administrator_Guide/b_ScanCenter_Administrator_Guide_chapter_011110.html)
```

Comment:
please also replace the link with https://www.cisco.com/c/en/us/td/docs/security/cognitive/cognitive-intelligence-user-guide/m_dashboard.html 
It's also a little bit out of sync but better than the old one. I wouldn't mention old ScanCenter_Administrator_Guide at all.